improc/photometry.py

```diff
@@ -130,17 +131,22 @@ def get_image(self, dx, dy):
         else:
             im[iy:, :] = 0
 
+        if self.imsize != self.datasize:  # crop the image to the correct size
+            im = im[
+                (self.datasize - self.imsize) // 2 : (self.datasize + self.imsize) // 2,
+                (self.datasize - self.imsize) // 2 : (self.datasize + self.imsize) // 2,
+            ]
+
```

Comment:
I've been using fairly small cutouts (25 pixels) and some of the default apertures are really big. I think we talked about changing them.

Reply:
Hmm.  This is pretty serious downside on doing photometry on cutouts rather than directly on the sub image.  To really do it may require cutouts to be bigger than we would otherwise want them to be.

Reply:
I don't really see why we need to have such large apertures.